lib/swaps/Swaps.ts

```diff
@@ -214,44 +214,67 @@ class Swaps extends EventEmitter {
       throw new Error(`${currency} client's pubKey not found for peer ${peerPubKey}`);
     }
     req.setPubKey(pubKey);
-    const routes = (await client.queryRoutes(req)).getRoutesList();
-    this.logger.debug(`got ${routes.length} routes to destination: ${routes}`);
-    return routes;
+    try {
+      const routes = (await client.queryRoutes(req)).getRoutesList();
+      this.logger.debug(`got ${routes.length} routes to destination: ${routes}`);
+      return routes;
+    } catch (err) {
+      this.logger.debug(`error calling getRoutes for ${currency} to ${pubKey}: ${JSON.stringify(err)}`);
+      if (typeof err.message === 'string' && err.message.includes('unable to find a path to destination')) {
```

Comment:
You're relying on `unable to find a path to destination`, but I encountered also `target not found` (might be others as well).

Reply:
Thanks, I'll add that case as well. If we encounter others we can add those.